packages/manager/apps/dedicated/client/app/dedicated/server/server.service.js

```diff
@@ -0,0 +1,2406 @@
+import assign from 'lodash/assign';
+import camelCase from 'lodash/camelCase';
+import compact from 'lodash/compact';
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import head from 'lodash/head';
+import indexOf from 'lodash/indexOf';
+import map from 'lodash/map';
+import parseInt from 'lodash/parseInt';
+import set from 'lodash/set';
+import snakeCase from 'lodash/snakeCase';
+import sortBy from 'lodash/sortBy';
+import uniq from 'lodash/uniq';
+
+export default class ServerF {
```

Comment:
**nit**: weird class name no ?

Reply:
@JeremyDec I think it was done that way due to a `Server` service already existing